<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDroneDesktop</class>
 <widget class="QMainWindow" name="QDroneDesktop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1269</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="wUpperContainer" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>344</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="wVideoContainer">
         <property name="minimumSize">
          <size>
           <width>380</width>
           <height>340</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="teCoding"/>
       </item>
       <item>
        <widget class="QFrame" name="wCtrlContainer">
         <property name="minimumSize">
          <size>
           <width>340</width>
           <height>340</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1249</width>
         <height>372</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QListView" name="listView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1269</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDrone">
    <property name="title">
     <string>Drone</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisonnect"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuDashboard">
    <property name="title">
     <string>Dashboard</string>
    </property>
    <addaction name="actionStart_Updating"/>
    <addaction name="actionStop_Updating"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDrone"/>
   <addaction name="menuDashboard"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConnect">
   <property name="text">
    <string>&amp;Connect</string>
   </property>
  </action>
  <action name="actionDisonnect">
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionDiscover">
   <property name="text">
    <string>&amp;Discover</string>
   </property>
  </action>
  <action name="actionStart_Updating">
   <property name="text">
    <string>Start Updating</string>
   </property>
  </action>
  <action name="actionStop_Updating">
   <property name="text">
    <string>Stop Updating</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
